rest/CMakeLists.txt

```diff
@@ -27,7 +33,7 @@ etn_target(shared ${PROJECT_NAME}-rest
 etn_configure_file(
     conf/42_discovery.xml.in
 
-    TARGET      ${PROJECT_NAME}-rest
+    TARGET      ${PROJECT_NAME}
```

Comment:
this taget is ${PROJECT_NAME}-rest
WHy do you attach this config to different target?